frontend/public/lib/imageOperations.js

```diff
@@ -2,7 +2,15 @@ const DEVELOPMENT = ["development", "develop", "test"].includes(process.env.ENVI
 import imageCompression from "browser-image-compression";
 
 export function getImageUrl(url) {
+
+  var IsWebPCompatibleBrowser = false;
   if (!url) return;
+
+
+  if (IsWebPCompatibleBrowser===true) {
+    url = url.replace('.jpeg','.webp')
+  }
+  console.log(url);
```

Comment:
Nit: just some minor rewrite following more modern JS best practices!
```suggestion
  let isWebPCompatibleBrowser = false;
  if (!url) return;


  if (isWebPCompatibleBrowser) {
    url = url.replace('.jpeg', '.webp');
  }
```